### ISSUE TITLE
I could eventually accomplish to build project but :( 

### ISSUE THREAD

**Issue by MehmetAran** [KoMaR1911/C4USMultiHack-Metin2#21]
Hi Komar, 
I am a player in Turkish server, I built project and got a dll which name is "EngineX-Kevra.dll".  Then I injected this dll to the game. But hack user interface didn't appear in the game. I struggle to solve problem but i couldn't. So I know you are busy but I asked you about your help. Thank you so much.

**Comment by KoMaR1911** [comment]
return true on hwid check if still doesnt works you did something wrong

**Comment by akinqenc** [comment]
syntax hatalarını nasıl çözdün?

**Comment by akinqenc** [comment]
https://i.imgur.com/El7hJkx.png
I am getting error "Cheat Wrong Version."
MehmetAran, do you also encounter that error?

**Comment by MehmetAran** [comment]
Hi bro, @akinqenc 
I downloaded Directx SDK 8 and include it to visual studio project, then i ran release mode to compile. A error occurred when i ran it. The error is about a function related SYSTEMCALL, the function had to return a result, i add return value as NULL.If i don't remember wrongly, three errors occurred in Global.h file. I deleted null values like that 0x2303F + null. I deleted " + null"  section. But i could not run the hack in the game.

**Comment by akinqenc** [comment]
@MehmetAran Use debug mode to compile. Nothing happens in release mode. I guess debug mode on .dll shared before. The created size is high.

**Comment by kanamarea31** [comment]
@KoMaR1911 hi buddy. i have too this error. please man only say why

**Comment by InfamousMaaffieeeaa** [comment]
@akinqenc what version of vs studio do you use ?

**Comment by akinqenc** [comment]
vs2019

**Comment by InfamousMaaffieeeaa** [comment]
@akinqenc i'm getting an error at the end of the compile.
"Windows cannot find ..\Protect.bat"
And after that i'm getting a dll 3.1MB that dosen't inject
you have any ideea on how i can fix it ?

**Comment by akinqenc** [comment]
use debug build

**Comment by MehmetAran** [comment]
Hi @akinqenc , 
Can you build project correctly in debug mode?
I encountered some errors when i built it in debug mode.

**Comment by InfamousMaaffieeeaa** [comment]
> Hi @akinqenc ,
> Can you build project correctly in debug mode?
> I encountered some errors when i built it in debug mode.

yeah 46 errors :))

**Comment by kanamarea31** [comment]
Arkadaşlar 46 hatayı çözüyorum. pointerleri güncelliyorum ama dllyi inject ettiğimde oyun kapanıyor ve x::hook; kodunu çalıştırdığında oyun kapanıyor (dx9 eksik sanırım) çözebilen var mı @akinqenc @MehmetAran

**Comment by ghost** [comment]
> Merhaba Komar,
> Ben Türk sunucu bir oyuncuyum, proje inşa ettim ve adı "EngineX-Kevra.dll" olan bir dll aldım. Sonra bu dll'yi oyuna enjekte ettim. Ancak hack kullanıcı arayüzü oyunda görünmedi. Sorunu çözmek için mücadele ediyorum ama yapamadım. Meşgul olduğunu biliyorum ama sana yardımını sordum. Çok teşekkür ederim.
@MehmetAran 
Dostum bende buildliyorum hatasız. Gerekli ayarları yapıp Release'den. 
Protect ten pack açılıp şifreliyor.
injectliyorum menü arayüzü gelmiyor ama canavar gibi kurcalıyorum istersen discroddan konuşalım ?

**Comment by ghost** [comment]
@akinqenc discorddan ekler misin vs15 ve 19 dan her türlü buildi yapıyorum menü gelmiyor, menünün gelmeme sorunun azbuçuk çözdüm sayılır ama yapamadım.

**Comment by kanamarea31** [comment]
Reisler çözerseniz bana da söyler misiniz.

**Comment by vlad-dieaconu** [comment]
@Kronolu add me on discord please higg#6280

**Comment by vlad-dieaconu** [comment]
@InfamousMaaffieeeaa can you add me on discord please higg#6280

**Comment by kanamarea31** [comment]
> MehmetAran

selamlar hocam son durum nedir acaba ?

**Comment by MehmetAran** [comment]
Hi @kanamarea31 , I can speak Turkish but i have to write english because i want people which live in other countries also understand me. I spent a lot of time to understand structure of project. But i couldn't run it. So i think maybe @KoMaR1911 prepare a video or article for us to teach how we can compile this project, how we can find constant pointers etc.

**Comment by kanamarea31** [comment]
OK. but @KoMaR1911 is selfish. because we are coders should help us if he is bored he can help us by making a video! @MehmetAran can you add me discord. i tell one thing you Atilla#4540

**Comment by Kairixx** [comment]
stop begging for this cheat you animals  , lol

**Comment by 8xzibit8** [comment]
Hi, I have found some small Turkey youtuber that provides download link to metin 2 multihack...
The thing is, that he downloaded the files from here and successfully compiled the files to .dll. ( I don´t know if he did it himself or just downloaded it from someone else ).
So what I wanted to say ? Well, I have downloaded the .dll from his link and I could use your coding skills. I will send you that .dll so you can check the code in visual studio if it is safe without any added harmfull commands and learn from it so maybe we will understand how he compiled it.
@MehmetAran and @akinqenc if you guys are interested please pm me or something, I don´t know how it works here...

P.S: name of the youtuber is: BTW KeyCheck 2 ( he seems to be legit ).

Also I wanted to say that it is really shame that KoMaR1911 will not help us how to compile the file.
Lots of people that are good in coding are stealing his source and making this cheat paid or full of viruses like realitycheats. They don´t even credit him and calling themselves to be the authors...

**Comment by PadureanDaniel** [comment]
I get 2 errors when i'm trying to inject the dll, one is the injection method used returned NULL
or on manual map i get unable to resolve path for dependency: VMProtectSDK32.dll

**Comment by cosmin2205** [comment]
yep, i'm also here with that dll that doesn't inject and "Windows cannot find ..\Protect.bat" ... have you guys found any solution ?

**Comment by cosmin2205** [comment]
has anyone found the solution for cheat wrong version ? where should i modify that hwid check ?

**Comment by cosmin2205** [comment]
has anyone found the solution for cheat wrong version ?

**Comment by JulianRue** [comment]
@MehmetAran can you provide the directx sdk 8 download link ? I cant find it online :(
